src/VirtualClient/VirtualClient.Actions/ApacheBench/ApacheBenchExecutor.cs

```diff
@@ -0,0 +1,277 @@
+    {
+        private IPackageManager packageManager;
+        private IFileSystem fileSystem;
+        private ProcessManager processManager;
+        private ISystemManagement systemManagement;
+        private IStateManager stateManager;
+
+        /// <summary>
+        /// The ApacheBench workload executor
+        /// </summary>
+        /// <param name="dependencies">Provides all of the required dependencies to the Virtual Client component.</param>
+        /// <param name="parameters">
+        /// Parameters defined in the execution profile or supplied to the Virtual Client on the command line.
+        /// </param>
+        public ApacheBenchExecutor(IServiceCollection dependencies, IDictionary<string, IConvertible> parameters = null)
+            : base(dependencies, parameters)
+        {
+            this.systemManagement = dependencies.GetService<ISystemManagement>();
+            this.fileSystem = dependencies.GetService<IFileSystem>();
+            this.packageManager = dependencies.GetService<IPackageManager>();
+            this.processManager = this.systemManagement.ProcessManager;
+            this.SystemManagement = dependencies.GetService<ISystemManagement>();
+            this.stateManager = this.systemManagement.StateManager;
+        }
+
+        /// <summary>
+        /// Parameter defines the command line arguments to pass to the workload executable.
+        /// </summary>
+        public string CommandLine
+        {
+            get
+            {
+                return $"-k -n {this.NoOfRequests} -c {this.NoOfConcurrentRequests} http://localhost:80/";
+            }
+        }
+
+        /// <summary>
+        /// Allows overwrite to ApacheBench param for number of requests. 
+        /// </summary>
+        public int NoOfRequests
+        {
+            get
+            {
+                int noOfRequests = 50000;
+
+                if (this.Parameters.TryGetValue(nameof(this.NoOfRequests), out IConvertible value) && value != null)
+                {
+                    noOfRequests = value.ToInt32(CultureInfo.InvariantCulture);
+                }
+
+                return noOfRequests;
+            }
+        }
+
+        /// <summary>
+        /// Allows overwrite to ApacheBench param for number of requests. 
+        /// </summary>
+        public int NoOfConcurrentRequests
+        {
+            get
+            {
+                int noOfConcurrentRequests = 50;
+
+                if (this.Parameters.TryGetValue(nameof(this.NoOfConcurrentRequests), out IConvertible value) && value != null)
+                {
+                    noOfConcurrentRequests = value.ToInt32(CultureInfo.InvariantCulture);
+                }
+
+                return noOfConcurrentRequests;
+            }
+        }
+
+        /// <summary>
+        /// Provides methods for managing system requirements.
+        /// </summary>
+        protected ISystemManagement SystemManagement { get; }
```

Comment:
systemManagement is already declared initially as private. We can remove this one if not being used.